west-sbom-report.md

```diff
@@ -0,0 +1,104 @@
+> ### WARNING!
+> Some of the files does not contains license information or the license contained in them cannot be detected.<br>
+See [license details](internal_link) for list of files.
+Evaluate them manually to get license information.
+
+
+# Summary
+
+This report was generated from the following inputs:
+* `zephyr/zephyr.elf from build directory /home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test`
+* `files: **/*.c, **/*.h (relative to /home/doki/work/ncs2/nrf/samples/crypto/ecdsa/)`
+* `list of files read from: /home/doki/work/ncs2/sample-list.txt`
+
+Detected files are covered by the licenses:
+* [LicenseRef-Nordic-5-Clause OR Apache-2.0](internal_link)
+* [LicenseRef-Nordic-5-Clause](internal_link)
+* [Apache-2.0](internal_link)
+* Possibly other licenses that [cannot be detected automatically](internal_link)
```

Comment:
```suggestion
* [Other licenses](internal_link) that cannot be detected automatically
```

Reply:
I added there `Possibly` because the list of files may not contain other licenses, we simply don't know that, because it cannot be detected.